lago/providers/libvirt/vm.py

```diff
@@ -614,14 +615,36 @@ def _libvirt_xml(self):
                 device=disk_device,
             )
 
-            disk.append(
-                ET.Element(
-                    'driver',
-                    name='qemu',
-                    type=dev_spec['format'],
-                    discard='unmap',
-                ),
-            )
+            if bus == 'virtio':
+                disk.append(
+                    ET.Element(
+                        'driver',
+                        name='qemu',
+                        type=dev_spec['format'],
+                        discard='unmap',
+                        cache='writeback',
+                        iothread='1',
```

Comment:
Currently, we define `1` iothread per domain (it's hardcoded in the domain XML), do you think that we need to change it? or make it dynamic?